cli/pack/app_dir_test.go

```diff
@@ -0,0 +1,32 @@
+package pack
+
+import (
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/tarantool/tt/cli/pack/test_helpers"
+	"path/filepath"
+	"testing"
+)
+
+func TestCreateAppDir(t *testing.T) {
+	testPackageDir := t.TempDir()
+	testBundleDir := t.TempDir()
+	dirsToCreate := []string{
+		"bundle",
+	}
+	filesToCreate := []string{
+		"bundle/app1.lua",
+	}
+
+	err := test_helpers.CreateDirs(testBundleDir, dirsToCreate)
+	require.Nilf(t, err, "failed to create test dirs: %v", err)
```

Comment:
Why Nilf? There is a special API for check for error/no-error cases: NoError, Error, EqualError, etc. Please, use them.